test/Dialect/FIRRTL/failConstProp.mlir

```diff
@@ -0,0 +1,90 @@
+// RUN: circt-opt -pass-pipeline='firrtl.circuit(firrtl-imconstprop)' --split-input-file  %s | FileCheck %s
+
+; XFAIL: *
```

Comment:
Please add a comment explaining that these are xfail because they aren't implemented yet, not because they are suppose to fail.